git/git_intro.qmd

```diff
@@ -23,6 +23,29 @@ git clone https://github.com/user/repo
 
 ## File staging and git commit
 
+Staging in Git involves adding new, modified, or deleted files to a staging area before committing them. This allows for flexibility in choosing the files to commit.
+
+1. Check status via `git status` You'll see what branch you are on and status of files (untracked, modified, or deleted).  
+2. Stage Files to Prepare for Commit
+ - Stage all files: `git add .`
+ - Stage a file: `git add example.html`
+ - Stage a folder: `git add myfolder`
+3. Check status again: `git status` You should see there are changes ready to be committed.
+4. Unstage a File
+  - If you accidental stage something, use the following command to unstage it: `git reset HEAD example.html`
+5. Deleting Files
+  - If you delete files they will appear in git status as deleted, and you must use git add to stage them. Another way to do this is using git rm command, which both deletes a file and stages it all with one command:
+  - `git rm example.html` to remove a file (and stage it)
+  - `git rm -r myfolder` to remove a folder (and stage it)
+6. Commit Files 
+  -  `git commit -m "Message that describes what this change does"`
```

Comment:
Suggestion: Also add in a point on `git commit --amend` here to fix commit messages, or add new staged files to the previous commit.